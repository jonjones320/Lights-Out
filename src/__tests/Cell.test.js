import React from "react";
import { render } from "@testing-library/react";
import Cell from "./Cell";

it("renders Cell without crashing", function() {
  render(<Cell />);
});

it("matches Cell snapshot", function() {
  const { asFragment } = render(<Cell />);
  expect(asFragment()).toMatchSnapshot();
});