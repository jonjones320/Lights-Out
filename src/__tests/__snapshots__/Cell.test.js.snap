// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Cell snapshot 1`] = `<DocumentFragment />`;
